### DISCUSSION TITLE
Authentication providers list is shown on the login screen when there is only one active authentication strategy (e.g. LDAP), other than local

### DISCUSSION THREAD

**Discussion by madodig** [requarks/wiki#3551] [Error / Bug Report]
**Describe the bug**
If LDAP is the only active authentication strategy (other than local), and 'Hide Local Authentication Provider' is set, the Authentication providers list on the login screen is still shown, with LDAP being the only provider in the list.
If local database is the only active authentication strategy, the Authentication providers list on the login screen is hidden, regardless of 'Hide Local Authentication Provider' setting.

**To Reproduce**
1. Navigate to Administration / Authentication and make sure only LDAP strategy is active (local can't be disabled), click Apply;
2. Navigate to Administration / Security and select 'Hide Local Authentication Provider', click Apply;
3. Navigate to login page
    > Authentication providers list is shown on the login screen with LDAP being the only provider in the list.
4. Navigate to Administration / Authentication and disable LDAP strategy, click Apply
5. Navigate to login page
    > Authentication providers list is not shown on the login screen when local database is the only active strategy.

**Expected behavior**
Authentication providers list on the login screen is hidden if LDAP (or any other strategy that does not need to be redirected) is the only provider in the list, as it is the case when local database is the only active strategy.

**Host Info**
OS: CentOS Linux release 8.2.2004 (Core)
Wiki.js version: 2.5.136
Database engine: PostgreSQL 12.1

**Additional context**
This should also apply to other strategies (if introduced in the future) that ask for credentials (not being redirected).
Regarding strategies that need to be redirected to, behavior is expected, since user has to choose to be redirected if 'Bypass Login Screen' is not set.

**Comment by Sandst4** [comment]
Thanks for pointing out this issue. I found out that pull request #3102 hasn't been implemented yet and the issue still exists. Does anyone know how or when this will be fixed? Thanks!

**Comment by FarmDoc** [comment]
Also annoying, that even if LDAP is the only visible strategy, you still have to select it. I had users not selecting it and getting errors.
